docs/api/tutorials_intro.md

```diff
@@ -238,6 +237,6 @@ A [tutorial for creating and using a bootloader](../tutorials/bootloader.html),
 </tbody>
 </table>
 
-## Migrating to Mbed OS 5
+## Mbed OS 2 to bare metal
```

Comment:
```suggestion
## Migrating from Mbed 2 to bare metal
```
It was called Mbed 2 not Mbed OS 2. There was no OS it was a collection of libraries.